{"uid":"9c21120e657751fd","name":"Валидация: создание борды с именем — ожидаемый статус 400","fullName":"tests.test_backend.board.board_validation_test#test_create_board_name_validation","historyId":"74fc1fc8b34508057004265b516f98d8","time":{"start":1749744317928,"stop":1749744318167,"duration":239},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":true,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1749744310971,"stop":1749744310971,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"base_url","time":{"start":1749744310971,"stop":1749744310971,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"pytestconfig","time":{"start":1749744310972,"stop":1749744310972,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"delete_output_dir","time":{"start":1749744310972,"stop":1749744310972,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"owner_client","time":{"start":1749744310972,"stop":1749744311581,"duration":609},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_space","time":{"start":1749744311581,"stop":1749744312659,"duration":1078},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"temp_project","time":{"start":1749744312659,"stop":1749744316443,"duration":3784},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","steps":[{"name":"Проверка, что API вернул статус 400 при length > 50","time":{"start":1749744318167,"stop":1749744318167,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[{"uid":"4e7cdc050af4e3d6","name":"stdout","source":"4e7cdc050af4e3d6.txt","type":"text/plain","size":78}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":1,"hasContent":true},"afterStages":[{"name":"temp_space::0","time":{"start":1749744347725,"stop":1749744348325,"duration":600},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"backend"},{"name":"parentSuite","value":"tests.test_backend.board"},{"name":"suite","value":"board_validation_test"},{"name":"host","value":"pkrvmxyh4eaekms"},{"name":"thread","value":"2546-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_backend.board.board_validation_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"expected_status","value":"400"},{"name":"name","value":"'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":7,"skipped":0,"passed":2,"unknown":0,"total":9},"items":[{"uid":"ff45d2e3f299e8a6","reportUrl":"https://vaizcom.github.io/autotests/123//#testresult/ff45d2e3f299e8a6","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f6ed7455e50>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749744002591,"stop":1749744002591,"duration":0}},{"uid":"25e84d3d4b8afb4a","reportUrl":"https://vaizcom.github.io/autotests/122//#testresult/25e84d3d4b8afb4a","status":"passed","time":{"start":1749743621470,"stop":1749743622239,"duration":769}},{"uid":"4d8eb07c33d718f8","reportUrl":"https://vaizcom.github.io/autotests/119//#testresult/4d8eb07c33d718f8","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fc7b5913490>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742819738,"stop":1749742819738,"duration":0}},{"uid":"c7df673338b7de0a","reportUrl":"https://vaizcom.github.io/autotests/118//#testresult/c7df673338b7de0a","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f9cb88f1f10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742667106,"stop":1749742667106,"duration":0}},{"uid":"14d0dfc01cc64686","reportUrl":"https://vaizcom.github.io/autotests/117//#testresult/14d0dfc01cc64686","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f2b9f21f250>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749742496869,"stop":1749742496869,"duration":0}},{"uid":"ad94987fc7ee55b2","reportUrl":"https://vaizcom.github.io/autotests/116//#testresult/ad94987fc7ee55b2","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7efcc7f78ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749741233332,"stop":1749741233332,"duration":0}},{"uid":"39f5fa0eed1397b4","reportUrl":"https://vaizcom.github.io/autotests/115//#testresult/39f5fa0eed1397b4","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f28d1ddcd10>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740825792,"stop":1749740825792,"duration":0}},{"uid":"9a607d62ddb92e94","reportUrl":"https://vaizcom.github.io/autotests/114//#testresult/9a607d62ddb92e94","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.vaiz.dev', port=443): Max retries exceeded with url: /v4/Login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f271b110ad0>, 'Connection to api.vaiz.dev timed out. (connect timeout=None)'))","time":{"start":1749740652571,"stop":1749740652571,"duration":0}}]},"tags":["backend"]},"source":"9c21120e657751fd.json","parameterValues":["400","'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'"]}